### DISCUSSION TITLE
Issues with latest version of AUTOMATIC111

### DISCUSSION THREAD

**Discussion by ClaraSanders** [TheLastBen/fast-stable-diffusion#368] [Q&A]
I've been using this for some time now to merge models but today after the new inpainting update it doesn't work anymore. I get the error when trying to run the last cell:  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/webui.py", line 24, in <module>
    import modules.sd_vae
ModuleNotFoundError: No module named 'modules.sd_vae'
It is strange, do I have to upload a vae now to run it?

**Comment by TheLastBen** [comment]
you just need to update the repo

**Reply by josueversace** [reply]
I have the same error, and the repo is updated.

<img width="1118" alt="Captura de Pantalla 2022-11-06 a la(s) 2 04 14 p m" src="https://user-images.githubusercontent.com/56228552/200189984-af55cb0c-8df5-486a-bf68-d7002520e167.png">

**Reply by TheLastBen** [reply]
something is blocking your repo update, run the second cell again and copy the log
